api/src/main/java/com/datatorrent/api/DefaultInputPort.java

```diff
@@ -109,4 +109,8 @@ public void teardown()
    */
   public abstract void process(T tuple);
 
+  protected int incrementCount()
+  {
+    return ++this.count;
```

Comment:
Please omit `this` unless it is necessary.

Reply:
done